src/client/sandbox/event/unload.js

```diff
@@ -56,6 +56,22 @@ export default class UnloadSandbox extends SandboxBase {
         }
     }
 
+    _onDocumentClick (e) {
+        var target = e.target || e.srcElement;
+
+        if (target.tagName && target.tagName.toLowerCase() === 'a') {
+            var href = nativeMethods.getAttribute.call(target, 'href');
+
+            this.isFakeIEBeforeUnloadEvent = /(^javascript:)|(^mailto:)|(^tel:)|(^#)/.test(href);
```

Comment:
This could be any other pseudo-protocol. Shouldn't we check for `http/https` instead?